pkg/flexvolume/flexvolume.go

```diff
@@ -143,127 +149,128 @@ func processOpts(optsStr string) (Options, error) {
 
 // ExecDriver executes the appropriate FlexvolumeDriver command based on
 // recieved call-out.
-func ExecDriver(driver Driver, args []string) {
+func ExecDriver(logger *zap.SugaredLogger, driver Driver, args []string) {
 	if len(args) < 2 {
-		ExitWithResult(Fail("Expected at least one argument"))
+		ExitWithResult(logger, Fail(logger, "Expected at least one argument."))
 	}
 
-	log.Printf("'%s %s' called with %s", args[0], args[1], args[2:])
+	logger = logger.With("command", args[1])
+	logger.With("binary", args[0], "arguments", args[2:]).Debug("Exec called")
 
 	switch args[1] {
 	// <driver executable> init
 	case "init":
-		ExitWithResult(driver.Init())
+		ExitWithResult(logger, driver.Init(zap.New(nil).Sugar()))
```

Comment:
We need to be more careful with how we disable logs in `Init()`. This currently will disable error logs in `Init()` which isn't what we want. We just want to disable the success logs.